packages/features/calendars/weeklyview/components/Calendar.tsx

```diff
@@ -94,9 +73,22 @@ export function Calendar(props: CalendarComponentProps) {
                 />
                 <VeritcalLines days={days} />
 
+                <SchedulerColumns
```

Comment:
This was moved up from below, to make sure this lies underneath the hoverable event blocks.